OrbitCore/Pdb.cpp

```diff
@@ -79,8 +79,12 @@ FunctionInfo* Pdb::GetFunctionFromProgramCounter(uint64_t a_Address) {
   return it->second;
 }
 
-void Pdb::ApplyPreset(const PresetFile& preset) {
-  SCOPE_TIMER_LOG(absl::StrFormat("Pdb::ApplyPreset - %s", m_Name.c_str()));
+std::vector<FunctionInfo*> Pdb::FunctionsToSelect(
+    const PresetFile& preset) const {
+  SCOPE_TIMER_LOG(
```

Comment:
I would just remove this, it doesn't seem that relevant to time this.

Reply:
done.